### ISSUE TITLE
[a11y] Cannot choose a record set for a report/label

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1298]
**To Reproduce**
1. Go to Reports
2. Select a report or label
3. See undefined error

**Expected behavior**
It should display a list of record sets for the report to run in.

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/164466157-7feaacc1-66e8-47e8-9729-7b52e1cd49c3.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Edge

**Comment by maxxxxxdlp** [comment]
Duplicate of #1298
